CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 ### 4.0.0 (Next)
+* [#176](https://github.com/alexa-js/alexa-app/pull/176): Made params `expressApp` and `router` optional for `alexa.express()` - [@bgjehu](https://github.com/bgjehu).
```

Comment:
@dblock thanks for offering the words. I am not good with words and describing things! Will do the changes!